packages/react/src/components/form/NativeSelect/NativeSelect.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { forwardRef } from 'react';
+import type { ForwardedRef, ReactNode, SelectHTMLAttributes } from 'react';
+import cn from 'classnames';
+import { PadlockLockedFillIcon } from '@navikt/aksel-icons';
+
+import { omit } from '../../../utils';
+import { ErrorMessage, Label, Paragraph } from '../../Typography';
+
+import classes from './NativeSelect.module.css';
+import utilityClasses from './../../../utils/utility.module.css';
+import { useNativeSelect } from './useNativeSelect';
+
+export type NativeSelectProps = {
+  /**
+   * Specifies whether the select box is disabled.
+   * @default false
+   * */
+  disabled?: boolean;
```

Comment:
Suggest removing this to be consistent with other components, as we haven't documented other untouched native props